invenio_drafts_resources/services/records/config.py

```diff
@@ -132,6 +133,7 @@ class RecordServiceConfig(RecordServiceConfigBase):
     search_versions = SearchVersionsOptions
 
     components = [
+        DraftFilesComponent,  # Should this be here or just in tests?
```

Comment:
Comment: It's related to an issue that Audrius was working on. Currently, RDM-Records is the place where files are married to drafts, and thus 1) the component should actually live there instead or 2) we should move some of the RDM functionality to Drafts-Resources and provide a ``RecordWithFilesService`` or similar.

Reply:
Ok, I integrated `DraftFilesComponent` in the test config and am letting rdm-records integrate it.